package-lock.json

```diff
@@ -1,3 +1,1499 @@
 {
```

Comment:
Generally we don't check in `package-lock.json` for libraries. It can be removed.